### DISCUSSION TITLE
How do I install on Firefox Mobile?

### DISCUSSION THREAD

**Discussion by Starcr0ss** [dimdenGD/OldTwitter#948] [Q&A] [answered]
The readme says that you can install this extension on Firefox mobile but I do not see how this is possible now that the extension has been removed from the Firefox add-ons store. Is there some "dirty" method of installing it with a file instead of using add-on collections?

**Comment by dimdenGD** [comment, marked as answer]
Prob not possible since it got removed, use Kiwi Browser instead

**Reply by Starcr0ss** [reply]
Totally fair, I figured as much. Just wanted to confirm there wasn't some alternative method. Thank you!

**Comment by saoumfa** [comment]
There is a way to do it, I have been able to install dimden's Oldtweetdeck on Firefox Nightly on android.

go to settings in firefox, then about firefox and tap the logo icon 5 times to  enable debug menu
Set xpinstall.signatures.required to false in about:config
then from the settings menu you can now install an extension from file
